### PR TITLE
Release/snowplow mobile/0.4.0

### LINKED ISSUE #5: Add automated testing to mobile model
### Describe the feature
We should add automated tests to the repository such that on every PR the tests run to ensure that we are not making breaking changes.

### Who will this benefit?
This will benefit all developers of future versions of the mobile package, as well as all reviewers as there will be some checks in place to ensure that changes don't deviate from expected behaviour

### LINKED ISSUE #28: Add support for Databricks
### Describe the feature

Add support for the dbt mobile model for Databricks.

### PR DESCRIPTION
**Pull request by agnessnowplow**
## Description & motivation
Add automated testing to mobile model (Close #5)
Add support for Databricks (Close #28)

## Checklist
- [ ] I have verified that these changes work locally
- [ ] I have updated the README.md (if applicable)
- [ ] I have added tests & descriptions to my models (and macros if applicable)

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: README.md [hunk 1]
```diff
@@ -14,12 +14,13 @@ Please refer to the [doc site][snowplow-mobile-docs] for a full breakdown of the
...
+The snowplow-mobile v0.4.0 package currently supports BigQuery, Databricks, Redshift, Snowflake & Postgres.
 
-|                 Warehouse                |     dbt versions    | snowplow-mobile version |
-|:----------------------------------------:|:-------------------:|:--------------------:|
-| BigQuery, Redshift, Snowflake & Postgres | >=1.0.0 to <2.0.0   |         0.3.2        |
-| BigQuery, Redshift, Snowflake & Postgres | >=0.20.0 to <1.0.0  |         0.2.0        |
+|                         Warehouse                    |     dbt versions    | snowplow-mobile version |
+|:----------------------------------------------------:|:-------------------:|:--------------------:|
+| BigQuery, Databricks, Redshift, Snowflake & Postgres | >=1.0.0 to <2.0.0   |         0.4.0        |
+|       BigQuery, Redshift, Snowflake & Postgres       | >=1.0.0 to <2.0.0   |         0.3.2        |
```
Comment (emielver) [inline README.md:22]: Can remove this line

Comment (emielver) [inline README.md:22]: Can remove this line

File: integration_tests/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+```bash
+bash integration_tests.sh -d {warehouse}
+```
+
+Supported warehouses:
+
+- redshift
+- bigquery
+- snowflake
+- postgres
```
Comment (emielver) [inline integration_tests/README.md:22]: Could we add databricks to this?

**Review by emielver — COMMENTED** [review]
Just two small changes then all good I think

**Review by emielver — APPROVED** [review]
LGTM! Great job on this!!!

**Pull request merged**